chia/full_node/mempool_manager.py

```diff
@@ -461,13 +457,7 @@ async def validate_spend_bundle(
                 coin_id,
                 EligibilityAndAdditions(is_eligible_for_dedup=False, spend_additions=[], is_eligible_for_ff=False),
             )
-            # We can't just go with the eligible for fast forward flag, we need to validate
```

Comment:
This comment seems to be removed unintentionally.

Reply:
yeah, it was unintentional. But this comment is also truncated.

Reply:
it doesn't seem like a helpful comment. The following line of code says the same thing:
```
mark_as_fast_forward = eligibility_info.is_eligible_for_ff and supports_fast_forward(rust_coin_spend)
```
The more pertinent questions would, perhaps, be "Why" can't we just fast-forward if the flag is set? (the code doesn't explain that)